internal/nostr/nostr.go

```diff
@@ -373,32 +408,37 @@ func (svc *Service) handleSubscription(ctx context.Context, subscription *Subscr
 		filter.Until = &until
 	}
 
-	sub, err := relay.Subscribe(ctx, []nostr.Filter{filter})
-	if err != nil {
-		errorChan <- err
-		return
-	}
-	errorChan <- nil
-	go func(){
-		for event := range sub.Events {
-			svc.Logger.WithFields(logrus.Fields{
-				"eventId":   event.ID,
-				"eventKind": event.Kind,
-			}).Infof("received event on subscription %d", subscription.ID)
-			responseEvent := ResponseEvent{
-				SubscriptionId: subscription.ID,
-				NostrId: event.ID,
-				Content: event.Content,
-				RepliedAt: event.CreatedAt.Time(),
+	reconnecting := false
```

Comment:
I don't think we should have to track `reconnecting`

Reply:
It is done to make sure we don't send anything to the errorChan again, because it will block it during reconnection unless errorChan is consumed by someone (which doesn't happen during reconnection as we don't send JSON response)

Reply:
Doesn't apply now, we try every 5s